src/main/kotlin/com/workos/directorysync/models/User.kt

```diff
@@ -54,6 +54,14 @@ open class User
   @JsonProperty("last_name")
   open val lastName: String?,
 
+  @JvmField
+  @JsonProperty("created_at")
+  open val createdAt: String,
```

Comment:
Thanks for the updates! Could we enhance to test to ensure it gets populated as expected?

https://github.com/workos/workos-kotlin/blob/fb14ca2adc4329fcec8823c26e74bd5c7309986b/src/test/kotlin/com/workos/test/directorysync/DirectorySyncApiTest.kt#L230

Reply:
Done!